packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -56,20 +56,15 @@ function storeActiveAccount(account: Account) {
 }
 
 export async function setActiveAccount(account: Account): Promise<void> {
-  const community = app.activeChainId();
   try {
-    const response = await axios.post(`${SERVER_URL}/setDefaultRole`, {
-      address: account.address,
-      author_community_id: account.community.id,
-      community_id: community,
-      jwt: userStore.getState().jwt,
-      auth: true,
-    });
-
-    if (response.data.status !== 'Success') {
-      throw Error(`Unsuccessful status: ${response.status}`);
-    }
-
+    await axios.post(
```

Comment:
Why not trpc?

Reply:
some of the remaining routes will need some more UI work, like creating hooks and cleanup... we can do next